fdbserver/BlobWorker.actor.cpp

```diff
@@ -520,6 +540,8 @@ ACTOR Future<Void> blobGranuleUpdateFiles(BlobWorkerData* bwData, Reference<Gran
 					metadata->currentDeltas = GranuleDeltas();
 					metadata->currentDeltaBytes = 0;
 
+					bwData->stats.mutationBytesBuffered = 0;
```

Comment:
Do we want to make this stat the current memory used by buffer or an accumulation over time? I think it's the latter (to parallel all the other stats) but wanted to confirm

Reply:
decrement mutationBytesBuffered by currentDeltaBytes